shared/src/test/scala/pl/metastack/metatime/TimeTest.scala

```diff
@@ -14,18 +14,33 @@ object TimeTest extends SimpleTestSuite with TestIgnored {
   }
 
   testIgnored("hour") {
-    assertEquals(Hour(25).days, Day(1 + 1.0 / 24))
+    assertEquals(Hour(25).days, Day(1 + 1 / 24))
     assertEquals(Hour(25).days.truncate, Day(1))
   }
 
-  testIgnored("operators") {
+  test("operators") {
     assertEquals(Time(23, 10) > Time(22, 0), true)
     assertEquals(Time(23, 10) > Hour(22), true)
-    assertEquals(5.hours - 1.minute, 4.hours + 59.minutes)
+    assertEquals(Hour(2) > Minute(80), true)
+  }
+
+  test("plusTime") {
+    assertEquals((Time(Hour(30) + Hour(20))).equals(Time(hour=50)), true)
+    assertEquals((Time(Minute(20) + Second(40))).equals(Time(hour=10, minute=20)), false)
+    assertEquals((Time(Hour(2) + Minute(60))).equals(Time(hour=3)), true)
+  }
+
+  test("minusTime") {
+    assertEquals((Time(Minute(50) - Minute(10))).equals(Time(minute=40)), true)
+    assertEquals((Time(Second(1) - Millisecond(100))).equals(Time(milliseconds=900)), true)
+    assertEquals((Time(Hour(50) - Minute(100))).equals(Time(hour=48, minute = 20)), true)
   }
 
   testIgnored("now") {
-    assertEquals(Time.now(), ???)
+    val timeBefore = Time.now()
+    //Add Delay
```

Comment:
You're right. In JavaScript you'll have to use `setTimeout`. As soon as you have implemented the scheduler, you can use `Scheduler.in` here. It may also be useful to have a function `toFuture` in `Task`. Otherwise, the test suite may terminate before the test case has completed.